client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/step-container.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+import FoldableCard from 'components/foldable-card';
+
+const StepContainer = ( { isSuccess, isWarning, isError, isProgress, title, summary, children, expanded, toggleStep, translate } ) => {
+	const getIcon = () => {
+		if ( isSuccess ) {
+			return 'checkmark-circle';
+		}
+		if ( isWarning ) {
+			return 'notice-outline';
+		}
+		if ( isError ) {
+			return 'notice-outline';
+		}
+		return '';
+	};
+	const className = classNames( {
+		'is-success': isSuccess,
+		'is-warning': isWarning,
+		'is-error': isError,
+	} );
+
+	summary = <span className={ className }><span>{ summary }</span>
+	<Gridicon icon={ getIcon() } className={ className } size={ 18 } /></span>;
+	const header = (
+		<div>
+			<div className="label-purchase-modal__step-status">
+				{ isProgress &&
+					<div className="label-purchase-modal__loading-spinner">
+						<Spinner />
```

Comment:
The "Step loading" spinner is displayed at the left of the card, but the status Gridicons (step has errors, step is OK) are at the right. Is this intentional? (@kellychoffman?)

Also the spinner is missaligned (see screenshot).

![gridicons](https://user-images.githubusercontent.com/1715800/31619438-6a3f6aaa-b28c-11e7-856b-e2b3d7510abc.png)